### PR TITLE
Added South Korea scraper.

### PR DESCRIPTION
**Pull request by jacobmcgowan**

### CODE REVIEW

**Comment by jacobmcgowan** [comment]
Found an issue with the reported deaths. Will fix and resubmit.

File: src/shared/scrapers/KOR/index.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+   * @returns {string} The common name of the province or the city name if a
+   * special city was given.
+   */
+  _parseProvince(officialEnglishName) {
+    return this._provinceNames[officialEnglishName] || officialEnglishName;
+  },
+  async scraper() {
+    const states = [];
+    const $ = await fetch.page(this.url);
+    const $table = $('table.num');
```
Comment (chunder) [inline src/shared/scrapers/KOR/index.js:68]: Can you test to be sure you found the table? 

`if ($table.length === 0) { throw new Error('Table not found'); }`

Comment (chunder) [reply src/shared/scrapers/KOR/index.js:68]: Can you also check each column header against your `fieldPositions` to be sure you throw an error if they change the table layout?

**Comment by lazd** [comment]
@qgolsteyn @chunder either of your CR this?

**Pull request merged**